openreview/venue_request/venue_request.py

```diff
@@ -715,18 +715,24 @@ def setup_decision_stage(self):
                 'required': True
             },
             'decision_options': {
-                'description': 'What are the decision options (provide comma separated values, e.g. Accept (Best Paper), Accept, Reject)? Leave empty for default options - "Accept (Oral)", "Accept (Poster)", "Reject"',
+                'description': 'List all decision options. By default, decisions containing "Accept" are considered Accept options. Otherwise, please specify the accept options in "Accept Decision Options". All other decisions are considered reject options. Provide comma separated values, e.g. "Accept (Best Paper), Invite to Archive, Reject". Leave empty for default options - "Accept (Oral)", "Accept (Poster)", "Reject"',
                 'value-regex': '.*',
                 'order': 30
             },
-            'make_decisions_public': {'description': 'Should the decisions be made public immediately upon posting? Default is "No, decisions should NOT be revealed publicly when they are posted".',
+            'accept_decision_options': {
+                'description': 'What are the accept decision options? If your accept options do not contain "Accept", please specify them here. Provide comma separated values, e.g. "Accept (Best Paper), Invite to Archive".',
```

Comment:
I know there is an example, but the wording of 

> If your accept options do not contain "Accept", please specify them here.

makes it sound like PCs should only specify those accept options that do not contain the word `Accept`, and it could be confusing. 

Maybe something like:

> What are the accept decision options? Please specify all decision options that signify acceptance to the venue. Any decision option not specified here will be treated as a rejection.